rachellegelden/Home-Together-Canada
# HTC-462: Search Criteria Component

# [Issue 462](https://github.com/rachellegelden/Home-Together-Canada/issues/462)

closes #462 

## Summary
- Created new Search Criteria Component with Container
- Created test for search criteria
- Created new generalUtils file, tests
- Updated AccountSummaryContainer, Updated MemberProfileSummary, Added new MockData
- Added new prop to Status for multi-select
- refactored func in memReg to use utility function
- Updated tests and snapshots

## Relevant Motivation & Context
- Feature, Code Clean up

## Testing Instructions
- Login as a Member
- Go to Search Criteria
- Ensure validation works as expected 
- Change MockData, ensure changes are displayed (you'll prob have to restart your react app after changing Mock Data)

## Developer checklist prior to opening this pull request:


## Reviewer
